au/code/au/apply_rational_magnitude_to_integral.hh

```diff
@@ -110,6 +110,12 @@ constexpr bool is_known_to_be_less_than_one(Magnitude<BPs...>) {
 template <typename T, typename MagT, bool IsMagLessThanOne>
 struct MaxNonOverflowingValueImplWhenNumFits;
 
+// Implementation helper for "the value is zero" (which recurs a bunch of times).
+template <typename T>
+struct ValueIsZero {
+    static constexpr T value() { return T{0}; }
+};
```

Comment:
This one tripped me up a bit since "is" to me usually means an API that answers the question "is this thing that?"  Is this just `ValueZero`?